pkg/storage/util_test.go

```diff
@@ -143,10 +146,23 @@ func (m *mockChunkStore) Get(ctx context.Context, userID string, from, through m
 }
 
 func (m *mockChunkStore) GetChunkRefs(ctx context.Context, userID string, from, through model.Time, matchers ...*labels.Matcher) ([][]chunk.Chunk, []*chunk.Fetcher, error) {
-	refs := make([]chunk.Chunk, 0, len(m.chunks))
-	// transform real chunks into ref chunks.
-	for _, c := range m.chunks {
-		r, err := chunk.ParseExternalKey("fake", c.ExternalKey())
+	var chunks [][]chunk.Chunk
+	var fetcher []*chunk.Fetcher
+	chunksWithFetcher := []int{4, 1, 2}
```

Comment:
This might need a small comment as to why.

Reply:
Sure. Will add